source/mir/combinatorics/package.d

```diff
@@ -427,7 +427,7 @@ in
 }
 body
 {
-    return CartesianPower(n, new size_t[repeat]);
+    return CartesianPower(cast(uint) n, new uint[cast(uint) repeat]);
```

Comment:
I need to cast `repeat` - as uint[size_t repeat]` is invalid - therefore casting at the same place made sense to me